src/chess/board.rs

```diff
@@ -17,10 +17,11 @@ pub struct Board {
     enp_sq: u8,
     rights: u8,
     halfm: u8,
+    ply: u16,
```

Comment:
Can you move this out of the board struct, as you mentioned it can be done entirely in the UCI code?

Reply:
Done in 0d45487